tests/test_integration.py

```diff
@@ -204,16 +202,6 @@ def do_request(self, req_num):
         # latency shouldn't be outside this range.
         assert 30 < duration_ms < 300
 
-        usage = telemetry["last_request_metrics"]["usage"]
```

Comment:
accidental?

Reply:
You're right, none of the changes in this file are relevant. Removing.